tensorflow/core/grappler/optimizers/data/noop_elimination_test.cc

```diff
@@ -43,6 +43,23 @@ NodeDef *MakeUnaryNode(StringPiece node_type, int count, string input_node,
                               GetCommonAttributes(), graph);
 }
 
+NodeDef *MakeShardNode(int count, string input_node, MutableGraphView *graph) {
```

Comment:
Instead of introducing `MakeShardNode`, generalize `MakeUnaryNode` to take a list of integers (as opposed to a single integer).

Reply:
Updated.